modules/toast/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,56 @@
+/// <reference path="../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {action} from '@storybook/addon-actions';
+import {ComponentStatesTable, permutateProps} from '../../../../utils/storybook';
+import {exclamationCircleIcon} from '@workday/canvas-system-icons-web';
+import {Toast} from '../index';
+
+const IconButtonStates = () => (
+  <React.Fragment>
+    <div>
+      <StaticStates>
+        <ComponentStatesTable
+          rowProps={[
+            {label: 'Default', props: {}},
+            {label: 'Custom Icon', props: {icon: exclamationCircleIcon}},
+            {
+              label: 'Custom Icon Color',
+              props: {icon: exclamationCircleIcon, iconColor: colors.cinnamon500},
+            },
+            {label: 'On Close', props: {onClose: action('close button clicked')}},
+            {
+              label: 'With Action Link',
+              props: {
+                onActionClick: action('action button clicked'),
+                actionText: 'View More Details',
+              },
+            },
+          ]}
+          columnProps={permutateProps({
+            Default: [{value: undefined, label: 'Default'}],
+          })}
```

Comment:
Nit: there's only one prop so you don't need to permutate
```suggestion
          columnProps={[
            {label: 'Default', props: {}}
          ]}
```